### ISSUE TITLE
Update-DbaInstance Reports that 13.0.6435 is the latest available

### ISSUE THREAD

**Issue by bwiggins10** [dataplat/dbatools#9563]
### Verified issue does not already exist?

I have searched and found no existing issue

### What error did you receive?

WARNING: [Get-SqlInstanceUpdate] 13.0.6435 on computer [Computer.Domain] is already the latest available

### Steps to Reproduce

```powershell
# provide your command(s) executed pertaining to dbatools
# please include variable values (redacted or fake if needed) for reference

$ServerList = "Server1", "Server2", "Server3"

$ServerList | foreach-object -parallel {
Update-DbaInstance -ComputerName $_ -Path C:\Temp -Download -Restart -Confirm:$false -Verbose
}
```

### Please confirm that you are running the most recent version of dbatools

2.1.28

### Other details or mentions

Trying to patch to 13.0.6455.2 - I can see that version is present in dbatools-buildref-index.json

### What PowerShell host was used when producing this error

PowerShell Core (pwsh.exe), Windows PowerShell (powershell.exe), Windows PowerShell ISE (powershell_ise.exe)

### PowerShell Host Version

Tested on both (In PS-5 I omitted the -parallel tag):
5.1.17763.6530

7.4.6

### SQL Server Edition and Build number

SQL 2016 - 13.0.6435.1

### .NET Framework Version

.NET Framework 4.8.4770.0

**Comment by niphlod** [comment]
There's an exact issue for the same exact "problem".
https://github.com/dataplat/dbatools/issues/9327

tl;dr: not a problem, it's that way by design. If you need to update your instances to something which is not AT LEAST a "CU", you need to use the -Kb parameter.

**Comment by niphlod** [comment]
closing this, feel free to reopen if needed
